src/DotLiquid.Tests/Helper.cs

```diff
@@ -51,15 +52,20 @@ public static void AssertTemplateResult(string expected, string template, INamin
             AssertTemplateResult(expected: expected, template: template, anonymousObject: null, namingConvention: namingConvention);
         }
 
-        public static void AssertTemplateResult(string expected, string template, Hash localVariables, SyntaxCompatibility syntax = SyntaxCompatibility.DotLiquid20)
+        public static void AssertTemplateResult(string expected, string template, Hash localVariables, IEnumerable<Type> localFilters, SyntaxCompatibility syntax = SyntaxCompatibility.DotLiquid20)
         {
             var parameters = new RenderParameters(System.Globalization.CultureInfo.CurrentCulture)
             {
                 LocalVariables = localVariables,
-                SyntaxCompatibilityLevel = syntax
+                SyntaxCompatibilityLevel = syntax,
+                Filters = localFilters
             };
             Assert.AreEqual(expected, Template.Parse(template).Render(parameters));
         }
+        public static void AssertTemplateResult(string expected, string template, Hash localVariables, SyntaxCompatibility syntax = SyntaxCompatibility.DotLiquid20)

```

Comment:
Need a new line above
```suggestion

        public static void AssertTemplateResult(string expected, string template, Hash localVariables, SyntaxCompatibility syntax = SyntaxCompatibility.DotLiquid20)
```